### ISSUE TITLE
tensorflow installation fails when building from dockerfile

### ISSUE THREAD

**Issue by doriclazar** [google/mediapipe#3446]
**Specs**
- OS Platform and Distribution: Windows 11 - Building Ubuntu 18.04 bionic
- Programming Language and version: Default - python 3.6.9
- MediaPipe version: main branch - HEAD

**The problem**
Dockerfile fails on a message:
protobuf requires Python '>=3.7' but the running Python is 3.6.9
this is caused because the tensorflow 1.14.0 requires protobuf>=3.6.1, instead of protobuf==3.6.1. Because of that, by default, it tries to install the latest protobuf (now 4.21.0) which requires python>=3.7.

**Reproduction**
Execute build from the Dockerfile:
`$ docker build .`

**Logs**:
```
Collecting protobuf>=3.6.1 (from tensorflow==1.14.0)
  Downloading https://files.pythonhosted.org/packages/6c/be/4e32d02bf08b8f76bf6e59f2a531690c1e4264530404501f3489ca975d9a/protobuf-4.21.0-py2.py3-none-any.whl (164kB)
    100% |################################| 174kB 8.1MB/s 
protobuf requires Python '>=3.7' but the running Python is 3.6.9
```

**Proposed solution**
adding a line 53 in [Dockerfile](https://github.com/google/mediapipe/blob/4a20e9909d55838d5630366ce719844cf06ae85c/Dockerfile):
`RUN pip3 install protobuf==3.6.1`
will solve it. I'll create a pull request with this change.

**Comment by doriclazar** [comment]
Fixed by #3473

**Comment by ritwikmishra** [comment]
I solved this by simply

```
pip install protobuf==3.6.1
```
